src/metabase/api/card.clj

```diff
@@ -166,15 +185,7 @@
   (let [raw-card (t2/select-one Card :id id)
         card (-> raw-card
                  api/read-check
-                 (t2/hydrate :creator
-                             :dashboard_count
-                             :parameter_usage_count
-                             :can_write
-                             :average_query_time
-                             :last_query_start
-                             [:collection :is_personal]
-                             [:moderation_reviews :moderator_details])
-                 collection.root/hydrate-root-collection
+                 hydrate-card-details
                  (cond-> ;; card
                    (:dataset raw-card) (t2/hydrate :persisted))
                  (last-edit/with-last-edit-info :card))]
```

Comment:
Why isn't `(last-edit/with-last-edit-info :card)` part of `hydrate-card-details`?

Reply:
I was wondering about this as well. It's the same situation in api.dashboard in terms of when to update the last edit info. It does collect the logic in one place, but it feels like it isn't part of the hydration logic. WDYT?